README.md

```diff
@@ -83,8 +117,8 @@ curl -v \
 curl -v \
      -X POST \
      -H 'Content-Type: application/json' \
-     -d '{ "email": "ryan.bard@mailinator.com", "firstName": "Ryan", "lastName": "Bard", "birthDate": 344390000000 }' \
-     'http://localhost:8080/api/users'
+     -d '{ "name": "Test Program", "organization": { "id": 1 }, "startYear": 2005, "streetAddress": "456 Peachtree St.", "state": "GA", "zipCode": "30332" }' \
+     'http://localhost:8080/api/programs'
```

Comment:
I'm not sure I understand the question.

If you are asking if we'll have basic CRUD of the Program (programs created via a POST) for right now (and maybe keep it later even after there is a request process), then yes :)